### ISSUE TITLE
Mobile responsiveness for better user experience.

### ISSUE THREAD

**Issue by neeraj542** [neeraj542/Personal-Finance-Tracker#3]
Enhancing mobile responsiveness for better user experience.

**Issue Description:**

- Improve layout for various screen sizes.
- Enhance touch-friendly interactions.
- Enhance user experience on mobile devices.

**Comment by Samaresh-Das** [comment]
Would like to work on this, I saw the page and some sections seems already responsive, shall I also touch those to give a proper 100% responsiveness?

**Comment by neeraj542** [comment]
Yes, You can work on this issue and also freely create new issues to maintaining better code.

**Comment by Samaresh-Das** [comment]
> Yes, You can work on this issue and also freely create new issues to maintaining better code.

Sure, thanks

**Comment by Samaresh-Das** [comment]
> Yes, You can work on this issue and also freely create new issues to maintaining better code.

Sure, thanks. BTW can you please assign it, just for the formalities

**Comment by Samaresh-Das** [comment]
@neeraj542 I did the changes, you can check it out. PR #20
